app/src/localization/en-CA.json

```diff
@@ -352,5 +352,62 @@
   "backend-message-account.not.found": "No account with the given account name was found.",
   "cart-clear-msg": "You are about to remove all items from",
   "cart-clear-confirmation": "These items will be permanently removed. Continue?",
-  "remove-all-cart-items": "Remove all cart items"
+  "remove-all-cart-items": "Remove all cart items",
+  "bundle.does.not.contain.min.constituents": "Bundle does not contain the minimum number of required bundle constituents.",
+  "cart.empty": "Shopping cart is empty.",
+  "need.billing.address": "Billing address must be specified.",
```

Comment:
Ideally we should bring out the messages properties to their own files to keep these localization files cleaner. We can have new ones named `messages-en-CA.json`, etc.